accountant_report_stakeholder_report/models/accountant_report_stakeholder_report.py

```diff
@@ -0,0 +1,264 @@
+# -*- coding: utf-8 -*-
+# Copyright 2022 OpenSynergy Indonesia
+# Copyright 2022 PT. Simetri Sinergi Indonesia
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from openerp import _, api, fields, models
+from openerp.exceptions import Warning as UserError
+
+
+class AccountantReportStakeholderReport(models.AbstractModel):
```

Comment:
```suggestion
class AccountantReportStakeholderReport(models.Model):
```

@Asrulbastian @mikevhe18 ini gw yg salah desain. Jgn dibuat jd abstract, model biasa aja. Nanti obt turunannya akan seperti taxform_pph_common